apps/frontend/src/main/ipc-handlers/ado/utils.ts

```
@@ -0,0 +1,156 @@
+}
+
+/**
+ * Sanitize a string for use in WIQL queries
+ * Prevents WIQL injection attacks
+ */
+export function sanitizeWiqlString(value: string): string {
+  if (!value) return '';
+  // Escape single quotes (WIQL string delimiter)
+  let sanitized = value.replace(/'/g, "''");
+  // Remove control characters
+  sanitized = sanitized.replace(/[\x00-\x1f\x7f-\x9f]/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)